SimplenoteTests/SPCredentialsTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+@testable import Simplenote
+
+
+// MARK: - SPCredentials Unit Tests
+//
+class SPCredentialsTests: XCTestCase {
+
+    /// Verifies that the new Credentials mechanism match the legacy values
+    ///
+    func testCredentialsMatchTheirLegacyCredentialsCounterpart() {
```

Comment:
The legacy credentials don't exist in CI, so this test can't pass – it only works on a local machine that already has the credentials present. IMHO we probably don't need it if we hand-verify both methods of credential import (from mobile secrets and from the template). WDYT?